src/raster/rasterband.rs

```diff
@@ -815,6 +815,16 @@ impl<'a> RasterBand<'a> {
             Ok(())
         }
     }
+
+    /// Fill this band with a constant value.
```

Comment:
Can you add a link to https://gdal.org/api/gdalrasterband_cpp.html#classGDALRasterBand_1a55bf20527df638dc48bf25e2ff26f353?

See the method above for an example.